src/sass/edu-benefits.scss

```diff
@@ -7,88 +7,20 @@
 @import "modules/m-schemaform";
 @import "modules/m-modal";
 @import "modules/m-omb-info";
+@import "modules/m-form-confirmation";
 
-$button-stroke: inset 0 0 0 1px;
```

Comment:
Great work @jbalboni, just want to make sure that some styles were deliberately removed rather than replaced, as indicated by a quick static comparison.